_includes/templates/download.html

```diff
@@ -74,14 +88,163 @@ <h2>{{ page.latestversion }} {{CURRENT_RELEASE}} <a type="application/rss+xml" h
+
+      <li><p>{{page.check_gpg_output}}</p>
+        <ol><li><p>{{page.line_starts_with}} <code>gpg: {{page.localized_gpg_good_sig}}</code></p></li>
+          <li><p>{{page.complete_line_saying}} <code>{{page.localized_gpg_primary_fingerprint}} {{SIGNING_KEY_FINGERPRINT_EXPLODED}}</code></p></li>
+        </ol></li>
+
+      <li><p>{{page.gpg_trust_warning}}</p></li>
+
+    </ol>
+  </details>
+  <details>
+    {% assign GPG = "gpg" %}
+    <summary><strong>{{page.macos_instructions}}</strong></summary>
+    <ol>
+      <li><p>{{page.download_release}}</p></li>
+
+      <li><p>{{page.download_checksums}} <a href="{{ PATH_PREFIX }}/SHA256SUMS.asc">SHA256SUMS.asc</a></p></li>
+
+      <li><p>{{page.cd_to_downloads}}</p>
+
+        <pre class="highlight"><code>{{page.cd_example_linux}}</code></pre>
+
+        </li>
+
+      <li><p>{{page.verify_download_checksum}}</p>
+
+        <pre class="highlight"><code>shasum -a 256 --check SHA256SUMS.asc</code></pre></li>
+
+      <li><p>{{page.checksum_warning_and_ok | replace, "$(SHASUMS_OK)", page.localized_checksum_ok}} <code>{{FILE_PREFIX}}{{site.data.binaries.macdmg}}: {{page.localized_checksum_ok}}</code></p></li>
+
+      <li><p>{{page.install_gpg}} <a
+      href="{{GPG_DOWNLOAD_URL}}">{{page.gpg_download_page}}</a></p></li>
+
+      <li><p>{{page.obtain_release_key}} <a href="{{RELEASE_KEY}}">laanwj-releases.asc</a></p>
+
+        <pre class="highlight"><code>gpg --recv-keys {{SIGNING_KEY_FINGERPRINT}}</code></pre></li>
+
+      <li>{{page.verify_checksums_file}}
+
+        <pre class="highlight"><code>gpg --verify SHA256SUMS.asc</code></pre></li>
+
+      <li><p>{{page.check_gpg_output}}</p>
+        <ol><li><p>{{page.line_starts_with}} <code>gpg: {{page.localized_gpg_good_sig}}</code></p></li>
+          <li><p>{{page.complete_line_saying}} <code>{{page.localized_gpg_primary_fingerprint}} {{SIGNING_KEY_FINGERPRINT_EXPLODED}}</code></p></li>
+        </ol></li>
+
+      <li><p>{{page.gpg_trust_warning}}</p></li>
+    </ol>
+  </details>
+
+  <details>
+    <summary><strong>{{page.linux_instructions}}</strong></summary>
+    <ol>
+      <li><p>{{page.download_release}}</p></li>
+
+      <li><p>{{page.download_checksums}} <a href="{{ PATH_PREFIX }}/SHA256SUMS.asc">SHA256SUMS.asc</a></p></li>
+
+      <li><p>{{page.cd_to_downloads}}</p>
+
+        <pre class="highlight"><code>{{page.cd_example_linux}}</code></pre>
+
+        </li>
+
+      <li><p>{{page.verify_download_checksum}}</p>
+
+        <pre class="highlight"><code>sha256sum --ignore-missing --check SHA256SUMS.asc</code></pre></li>
+
+      <li><p>{{page.checksum_warning_and_ok | replace, "$(SHASUMS_OK)", page.localized_checksum_ok}} <code>{{FILE_PREFIX}}-{{site.data.binaries.lin64}}: {{page.localized_checksum_ok}}</code></p></li>
+
+      <li><p>{{page.obtain_release_key}} <a href="{{RELEASE_KEY}}">laanwj-releases.asc</a></p>
+
+        <pre class="highlight"><code>gpg --recv-keys {{SIGNING_KEY_FINGERPRINT}}</code></pre></li>
```

Comment:
The output of this command is confusing, especially for a new gnupg user:

```
$ gpg --recv-keys 01EA5486DE18A882D4C2684590C8019E36C2E964
gpg: directory '/home/grok/.gnupg' created
gpg: keybox '/home/grok/.gnupg/pubring.kbx' created
gpg: /home/grok/.gnupg/trustdb.gpg: trustdb created
gpg: key 90C8019E36C2E964: public key "Wladimir J. van der Laan (Bitcoin Core binary release signing key) <laanwj@gmail.com>" imported
gpg: no ultimately trusted keys found
gpg: Total number processed: 1
gpg:               imported: 1
```

Maybe add something like "look for the line `key 90C8019E36C2E964: public key "Wladimir J. van der Laan (Bitcoin Core binary release signing key) <laanwj@gmail.com>" imported`"

Reply:
I went with,

> The output of the command above should say that at least one key was imported, updated, has new signatures, or remained unchanged.

In addition to being easier to update across translations if Wladimir changes keys or someone else takes over signing releases, the options of "updated," "new signatures," and "unchanged" matches what the user will see if they've previously imported the key.  E.g.,

```text
## With the key already on my keyring
$ gpg --recv-keys 01EA5486DE18A882D4C2684590C8019E36C2E964
gpg: key 90C8019E36C2E964: "Wladimir J. van der Laan (Bitcoin Core binary release signing key) <laanwj@gmail.com>" not changed
gpg: Total number processed: 1
gpg:              unchanged: 1

## Removed signatures with
##   gpg --edit-key 01EA5486DE18A882D4C2684590C8019E36C2E964 ; minimize ; save
$ gpg --recv-keys 01EA5486DE18A882D4C2684590C8019E36C2E964
gpg: key 90C8019E36C2E964: "Wladimir J. van der Laan (Bitcoin Core binary release signing key) <laanwj@gmail.com>" 47 new signatures
gpg: marginals needed: 3  completes needed: 1  trust model: pgp
gpg: depth: 0  valid:   1  signed:   0  trust: 0-, 0q, 0n, 0m, 0f, 1u
gpg: next trustdb check due at 2018-09-19
gpg: Total number processed: 1
gpg:         new signatures: 47

## I think "updated" is what's displayed when uid info changes
```

And it looks different enough from the failure case that there should hopefully be no confusion:

```text
$ gpg --recv-keys 01EA5486DE18A882D4C2684590C8019E36C2E965
gpg: keyserver receive failed: No data
```